src/transforms/basic.js

```diff
@@ -125,6 +125,10 @@ function sortValue(value) {
     channel = channel.slice(1);
     order = reverseOrder(order);
   }
+  if (typeof value === "string" && value?.startsWith("-")) {
+    value = value.slice(1);
+    order = reverseOrder(order);
+  }
```

Comment:
If *value* is a string here, it is a field name rather than a reducer or a channel name. Because field names come from the user’s data, they are much more diverse than channel names and reducer names (which are generally fixed by Plot, though we do allow “extra” channels for the tip mark). This would break field names that start with a hyphen, which is admittedly unlikely and uncommon, but still possible. And since we aren’t going to allow `x: "-foo"` for field names in general, this is somewhat inconsistent.

I don’t think we should introduce `sort: "-foo"` shorthand for field names. There are already three ways to do this, and that seems reasonable enough:

1. `sort: "foo", reverse: true`
2. `sort: {value: "foo", order: "descending"}`
3. `sort: (d) => -d.foo` (assuming foo is numeric)

We could introduce a value helper, perhaps: `sort: Plot.negate("foo")`. This would take the same arguments that Plot.valueof takes for *value*, but would return the negative value. It would only work for numbers, though, of course…

Or perhaps we could also introduce a sort helper: `sort: Plot.descending("foo")` which just returns `{value: "foo", order: "descending"}`.